BUILD.md

```diff
@@ -59,9 +55,9 @@ Runs after the top-level `yarn install`. This ensures `./packages/plexus` builds
 
 Note that `./packages/plexus` does not yet have any tests, as tracked in issue [#340](https://github.com/jaegertracing/jaeger-ui/issues/340).
 
-`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `yarn test` from its directory, rather than the repository root.
+`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `npm run test` from its directory, rather than the repository root.
```

Comment:
just `npm test` seems to work, right?
```suggestion
`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `npm test` from its directory, rather than the repository root.
```

Reply:
Yes, some npm commands are shortcuts to `npm run`.
`npm start` and `npm test` are shortcuts, but for example `npm build` is not a shortcut and must be written as `npm run build`.